cli/internal/run/dry_run.go

```diff
@@ -389,5 +400,11 @@ func (ht *taskSummary) toSinglePackageTask() singlePackageTaskSummary {
 		ResolvedTaskDefinition: ht.ResolvedTaskDefinition,
 		Framework:              ht.Framework,
 		ExpandedInputs:         ht.ExpandedInputs,
+		EnvVars:                ht.EnvVars,
 	}
 }
+
+type taskEnvVarSummary struct {
+	Configured []string `json:"configured"`
+	Inferred   []string `json:"inferred"`
+}
```

Comment:
This may also include global env vars that are affecting the task hash (i.e. `globalEnv` and the implicit turbo injected env var `THASH`)